Dockerfile

```diff
@@ -0,0 +1,31 @@
+# extend node alpine base image
+FROM node:8.4.0-alpine
```

Comment:
I think this could be switched to `8-alpine`. With `renovate`'s support for updating the digest value for a given tag, using the more general `8-alpine` tag will allow us to get updates whenever there are new versions of Node 8 released, while still getting the deterministic build behavior of using SHAs managed by `renovate`.

Reply:
I agree, this should provide a nice integration with the new Dockerfile renovation feature.  Thanks!